### DISCUSSION TITLE
2023-January Snapshot Notes

### DISCUSSION THREAD

**Discussion by rowanseymour** [nyaruka/rapidpro#4249] [Snapshots]
Hey everyone! 2023.01 is now the official stable snapshot and we've updated the recommended versions of all components in the [README](https://github.com/nyaruka/rapidpro#textit). This is our first release under [a BSL licence](https://github.com/nyaruka/rapidpro/blob/main/LICENSE). If you're not running a competitor to [textit.com](https://textit.com) then absolutely nothing changes. If you are, then we offer licencing options or you are welcome to continue with the [AGPL version](https://github.com/rapidpro/rapidpro).

This is also the first release to no longer support topups, and we've also cleaned up a lot of old legacy billing and branding code that didn't belong in the platform.

### New Features

 * Uploaded attachments for outgoing messages are now transcoded into different formats using ffmpeg, thumbnails are saved for images and videos.
 * Rewrite of channel logs so those are easier to create on the courier side and better displayed on the RapidPro side.
 * Incoming call triggers now support messaging flows - the call is hungup and the contact started in the messaging flow.
 * Scheduled messages can now be deleted properly.
 * Exports of messages, tickets and flow results now all use a common UI and support the same parameters (date range, contact group filtering).
 * Message attachments are deleted when messages are deleted in the UI.
 * Better controls for input of date values that show the timezone where appropriate.
 * Improved logging and reporting to users of channel specific errors.
 * Added JustCall channel type.
 * Better logging of celery scheduled tasks with their times and results.

### New Requirements

* Python 3.10 
* PostgreSQL 14
* Redis 6.2

It's possible that everything still works fine on the previous versions but it's only a matter of time before we add code that requires those newer versions.

**Comment by ericnewcomer** [comment]
Thanks for sharing @rowanseymour! Transcoded attachments is one of my favorite new features.
